TX.h

```diff
@@ -564,6 +715,76 @@ static inline void processSBUS(uint8_t c)
   lastSerialPPM = millis();
 }
 
+static inline uint16_t multiToPpm(uint16_t input) {
+  uint32_t value = input;
+  // Taken from https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Multiprotocol/Multiprotocol.h#L478-L483
+  value = (((value - 204) * 1000) / 1639) + 1000;
```

Comment:
That's correct, but OpenLRS's range is 800-2200 so total swing of 1400 instead of 1500. Then center adjustment is then also 800. :)